web/islands/PostControls.tsx

```diff
@@ -146,6 +146,26 @@ export function PostControls(props: PostControlsProps) {
       });
   }
 
+  const shareableExternally = navigator.clipboard != null ||
+    navigator.share != null;
+  function onShareExternally() {
+    if (navigator.share) {
+      navigator.share({
+        title: t("post.shareExternally.title", {
+          actor: `${post.actor.name}(${post.actor.handle})`,
+        }),
+        url: localPostUrl,
```

Comment:
The `localPostUrl` does not contain an absolute URL. It's a relative URL. Also, it does not refer to the actual permalink of the post, but the mirrored page of the post from Hackers' Pub when it's a remote post. I believe you should use `post.url ?? post.iri` instead here.

Reply:
I referenced non-fediverse applications to make this feature. I just noticed that sharing permalink of original post is proper in fediverse.